devtools-ci-index.yaml

```diff
@@ -2193,154 +2193,198 @@
             test_url: openshift.io
-            test_suite: quickstartTest
+            test_suite: analyticTest
+            cluster: cluster-one
             oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
             kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
             ee_test_start_time: '5 5-23/4 * * *'
             timeout: 60m
             disabled: true
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: openshift.io
-            test_suite: chequickstartTest
+            test_suite: cheTest
+            cluster: cluster-one
+            oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '5 6-23/4 * * *'
+            timeout: 60m
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: smokeTest
+            cluster: cluster-two
+            after: devtools-fabric8-test-build-master
+            oso_token_creds: e9a32e1e-0a38-4a96-896f-a266e1428cf9
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '5 4-23/4 * * *'
+            timeout: 60m
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: analyticTest
+            cluster: cluster-two
+            oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '5 5-23/4 * * *'
+            timeout: 60m
+            disabled: true
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: cheTest
+            cluster: cluster-two
             oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
             kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
             ee_test_start_time: '5 6-23/4 * * *'
             timeout: 60m
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: prod-preview.openshift.io
-            test_suite: runTest
+            test_suite: smokeTest
+            cluster: cluster-one
             after: devtools-fabric8-test-build-master
             oso_token_creds: 60ae2fd0-5ab0-4a12-bf75-6a81ab1b915f
             kc_refresh_token_creds: a75af77c-1137-4ea2-9690-968848b0334e
             ee_test_start_time: '5 4-23/4 * * *'
             timeout: 60m
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: prod-preview.openshift.io
-            test_suite: quickstartTest
+            test_suite: analyticTest
```

Comment:
Minor typo:
@ldimaggi want to change this to `analyticsTest` instead of `analyticTest`?

Reply:
I had intended to use the string "analytic"  ;-)

But - no worries - I just changed it to "analytics" - thanks!